.github/workflows/publish-release.yml

```diff
@@ -1,11 +1,6 @@
 name: Publish Extension
 
 on:
-  push:
-    branches:
-      - 'release'
-      - 'release/*'
-      - 'release-*'
   workflow_dispatch:
```

Comment:
Do we need `workflow_dispatch`
I added that as a way for manually publishing from CI.
I.e. if you run the CI pipline manually and add the message `true` in response to the message `Publish to VSCode marketplace?` then we publish it (i.e. its still automated on CI - via a click of a button).

Or awe we now saying that we'll always pull the extension VSIX down & upload to marketplace manually?
If thats the case we should remove this dispatch

Reply:
I'm assuming the answer is yes, else theres' no need to rename the `vsce_` command to `vsce`.
FYI - we might want to ensure the `VSC_TOKEN` has been added to github actions.

Reply:
VSC_TOKEN has been added. 

Not sure what you're saying otherwise. The workflow_dispatch would be the 'manual' upload. You just click on the github action to do it.

Reply:
Seems fine to me to have the option for workflow_dispatch. Would allow us to pop out a specific build early if we ever wanted to.